### PR TITLE
feat: Public note

### PR DESCRIPTION
**Pull request by kanopa**

### CODE REVIEW

File: Client/src/app/content/navigation/header/header.component.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ export class HeaderComponent implements OnInit, OnDestroy {
     this.destroy.complete();
   }
 
-  ngOnInit(): void {
+  async ngOnInit(): Promise<void> {
```
Comment (davydqq) [inline Client/src/app/content/navigation/header/header.component.ts:87]: The reason for the promise?

File: Client/src/styles.scss [hunk 2]
```diff
@@ -52,11 +52,20 @@ em {
   margin-right: 0.0625rem;
 }
 
-a:link, a:visited { // ONLY FOR INNER NOTE
```
Comment (davydqq) [inline Client/src/styles.scss:55]: the reason for removal?

File: Client/src/app/content/notes/full-note/directives/selection.directive.ts [hunk 3]
```diff
@@ -83,7 +86,9 @@ export class SelectionDirective implements OnDestroy, OnInit {
   }
 
   ngOnInit(): void {
-    this.initMouseHandlers();
+    if (!this.isReadonly) {
```
Comment (davydqq) [inline Client/src/app/content/notes/full-note/directives/selection.directive.ts:89]: Viewers have the opportunity to select text for copying. let`s discuss

File: Client/src/app/content/notes/full-note/full-note-components/collection-components/collection-photos/photo/photo.component.html [hunk 4]
```diff
@@ -9,6 +9,7 @@
     [@photoInit]="photo.loaded ? 'loaded' : 'noLoaded'"
     [src]="photo.photoFromBig"
     (load)="onLoadImage()"
+    alt="User album image"
```
Comment (davydqq) [inline Client/src/app/content/notes/full-note/full-note-components/collection-components/collection-photos/photo/photo.component.html:12]: need to support other langs

File: Client/src/app/core/signal-r.service.ts [hunk 5]
```diff
@@ -95,8 +95,8 @@ export class SignalRService {
     private readonly auth: AuthService,
   ) {}
 
-  init() {
-    this.startConnection();
+  async init() {
+    await this.startConnection();
```
Comment (davydqq) [inline Client/src/app/core/signal-r.service.ts:99]: Why is asynchrony necessary here?

File: Client/src/assets/locale/en.json [hunk 6]
```diff
@@ -64,7 +64,9 @@
       "dateAsc": "From old to new",
       "dateDesc": "From new to old",
       "order": "Custom draggable order"
-    }
+    },
+    "anonymous": "Anonymous",
```
Comment (davydqq) [inline Client/src/assets/locale/en.json:68]: translation for other langs

File: Client/src/app/content/notes/full-note/directives/selection.directive.ts (hunk 3 above)
Comment (kanopa) [reply Client/src/app/content/notes/full-note/directives/selection.directive.ts:89]: User can copy text, other features with copying available only on edited mode

File: Client/src/app/content/notes/full-note/full-note-components/collection-components/collection-photos/photo/photo.component.html (hunk 4 above)
Comment (kanopa) [reply Client/src/app/content/notes/full-note/full-note-components/collection-components/collection-photos/photo/photo.component.html:12]: About other langs, you show me tools which translate json to other languages. Lets dont translating to other language in every pull, go do it once before deployment

File: Client/src/styles.scss (hunk 2 above)
Comment (kanopa) [reply Client/src/styles.scss:55]: a:link looks like not working, and i not only remove, i add wrapper for this styles only applies to full note

**Pull request merged**